### DISCUSSION TITLE
Blue light filter

### DISCUSSION THREAD

**Discussion by donorport** [MonitorControl/MonitorControl#1351] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

You can only use more than one monitor using a DisplayLink-compatible docking station on Macbook Air M1 and the current bluelight solutions (f.lux & Iris) do not work on these external displays. 

### Describe the solution you'd like

A bluelight filter to go along with the brightness sliders

### Describe alternatives you've considered

N/A

**Comment by waydabber** [comment]
DisplayLink should have updated drivers to allow color temperature change. It should work both with f.lux and BetterDisplay. The relevant setting needs to be set in the DisplayLink app. If use betterdisplay, you also need to enable colortable manipulation for virtual displays under Settings/Displays.
